src/common/ImageAvatar.js

```diff
@@ -1,15 +1,24 @@
 /* @flow */
 import React, { PureComponent } from 'react';
-import { Image } from 'react-native';
+import { Image, StyleSheet } from 'react-native';
 
 import { nullFunction } from '../nullObjects';
 import { Touchable } from './';
+import UserStatusIndicator from '../common/UserStatusIndicator';
+
+const componentStyles = StyleSheet.create({
+  status: {
+    marginLeft: 21,
```

Comment:
That does not seem to work universally.
We might have different avatar sizes.
Can you think of a way to make it work for variable sizes?